src/pages/using-spark/guides/content-style.mdx

```diff
@@ -0,0 +1,127 @@
+---
+title: Content Style
+---
+
+import {
+  SprkStack,
+  SprkStackItem,
+  SprkButton,
+  SprkFlag,
+  SprkHeading,
+  SprkLink,
+  SprkTextInput,
+  SprkCard,
+  SprkIcon,
+  SprkDivider,
+} from '@sparkdesignsystem/spark-react';
+
+<h1 class="sprk-b-PageTitle sprk-b-TypeDisplayOne sprk-u-mbl sprk-u-Measure ">
+  Content Style
+</h1>
+
+UX Writers, designers and developers can use the **Content Style Guides** as a instruction to writing style, usage, text for copy, documentation, reference information, and copy for all Rocket Products digital experiences.
+
+<SprkDivider element="span" additionalClasses="sprk-u-mvn" />
+
+### Table Of Contents
+
+<ul>
+  <li>
+    <a href="#capitalization">Capitalization</a>
+  </li>
+  <ul class="sprk-b-List sprk-b-List--indented sprk-u-Measure sprk-u-mvs">
+    <li>
+      <a href="#sentence-case">Sentence Case</a>
+    </li>
+    <li>
+      <a href="#start-case">Start Case</a>
+    </li>
+  </ul>
```

Comment:
```suggestion
  <ul class="sprk-b-List sprk-b-List--indented sprk-u-Measure sprk-u-mvs">
    <li>
      <a href="#start-case">Start Case</a>
    </li>
    <li>
      <a href="#sentence-case">Sentence Case</a>
    </li>
  </ul>
```

These should be in the same order that they appear in the page.

Reply:
Thank you for catching this @bobwise , updated the TOC